src/platform/silabs/wifi/WifiInterfaceAbstraction.h

```diff
@@ -36,35 +37,41 @@
 #include "sl_si91x_types.h"
 #include "sl_wifi_constants.h"
 #include "sl_wifi_device.h"
+#endif // (SLI_SI91X_MCU_INTERFACE | EXP_BOARD)
 
-/*
- * Interface to RSI Sapis
- */
+/* Updated constants */
 
-#define WFX_RSI_DHCP_POLL_INTERVAL (250) /* Poll interval in ms for DHCP		*/
+constexpr size_t kWifiMacAddressLength = 6;
 
-// MAX SSID LENGTH excluding NULL character
-#define WFX_MAX_SSID_LENGTH (32)
+/* Defines to update */
+
+// TODO: Not sure why the pass key max length differs for the 917 SoC & NCP
+#if (SLI_SI91X_MCU_INTERFACE | EXP_BOARD)
 // MAX PASSKEY LENGTH including NULL character
 #define WFX_MAX_PASSKEY_LENGTH (SL_WIFI_MAX_PSK_LENGTH)
-#define SL_WIFI_ALLOCATE_COMMAND_BUFFER_WAIT_TIME_MS (1000)
 #else
-// MAX SSID LENGTH excluding NULL character
-#define WFX_MAX_SSID_LENGTH (32)
 // MAX PASSKEY LENGTH including NULL character
 #define WFX_MAX_PASSKEY_LENGTH (64)
```

Comment:
any idea where the 64 comes from?

Reply:
Wasn't able to find it quickly - don't know where it comes from.